src/Kernel.php

```diff
@@ -13,6 +13,11 @@
 {
     use MicroKernelTrait;
 
+    public function getProjectDir()
+    {
+        return __DIR__.'/..';
```

Comment:
Symfony guesses by finding a `composer.json`. Since it's not in the `fpm` image, we have to be explicit.

Reply:
It could be useful to have the `composer.lock` file in the image to check which package versions are available, but `vendor/composer/installed.json` already has all the information to track this